libtheoraenc.so needs to be linked to libtheoradec.so in order to avoid

symbol 'th_comment_query_count': can't resolve symbol in lib '/usr/lib/libtheoraenc.so.1'

when starting Freeswitch.

Patch downloaded from
http://www.sisyphus.ru/en/srpm/Sisyphus/libtheora/patches/0

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

--- libtheora/lib/Makefile.am
+++ libtheora/lib/Makefile.am
@@ -153,6 +153,7 @@ libtheoraenc_la_SOURCES = \
 libtheoraenc_la_LDFLAGS = \
   -version-info @THENC_LIB_CURRENT@:@THENC_LIB_REVISION@:@THENC_LIB_AGE@ \
   @THEORAENC_LDFLAGS@ $(OGG_LIBS)
+libtheoraenc_la_LIBADD = libtheoradec.la
 
 libtheora_la_SOURCES = \
 	$(decoder_sources) \
